sklearn/cluster/_optics.py

```diff
@@ -82,44 +82,44 @@ class OPTICS(ClusterMixin, BaseEstimator):
         See the documentation for scipy.spatial.distance for details on these
         metrics.
 
-    p : int, optional (default=2)
+    p : int, default=2
         Parameter for the Minkowski metric from
         :class:`sklearn.metrics.pairwise_distances`. When p = 1, this is
         equivalent to using manhattan_distance (l1), and euclidean_distance
         (l2) for p = 2. For arbitrary p, minkowski_distance (l_p) is used.
 
-    metric_params : dict, optional (default=None)
+    metric_params : dict, default=None
         Additional keyword arguments for the metric function.
 
-    cluster_method : str, optional (default='xi')
+    cluster_method : str, default='xi'
         The extraction method used to extract clusters using the calculated
         reachability and ordering. Possible values are "xi" and "dbscan".
 
-    eps : float, optional (default=None)
+    eps : float, default=None
         The maximum distance between two samples for one to be considered as
         in the neighborhood of the other. By default it assumes the same value
         as ``max_eps``.
         Used only when ``cluster_method='dbscan'``.
 
-    xi : float, between 0 and 1, optional (default=0.05)
+    xi : float, between 0 and 1, default=0.05
```

Comment:
Nit: For consistency with `min_samples`, let us change `float, between 0 and 1` by `float between 0 and 1`.

Reply:
ok